Build/windows/generate_vs17.bat

```diff
@@ -1,5 +1,5 @@
 :: Copyright(c) 2018 Intel Corporation 
 :: SPDX-License-Identifier: BSD-2-Clause-Patent
 
-cmake ../.. -G "Visual Studio 15 2017 Win64" -DCMAKE_ASM_NASM_COMPILER="yasm.exe"
+cmake ../.. "-GVisual Studio 15 2017 Win64" -DCMAKE_ASM_NASM_COMPILER="yasm.exe" -DCMAKE_CONFIGURATION_TYPES="Debug;Release"
```

Comment:
refer to:  https://stackoverflow.com/questions/24460486/cmake-build-type-not-being-used-in-cmakelists-txt